pkg/intermediate/types.go

```diff
@@ -33,11 +33,15 @@ type AggregationFlowRecord struct {
 	// inter-node flow and record from the node for the case of intra-node flow.
 	ReadyToSend               bool
 	waitForReadyToSendRetries int
-	// isMetaDataFilled is an indicator for IPFIX Mediator to check whether K8s
+	// areCorrelatedFieldsFilled is an indicator for IPFIX Mediator to check whether K8s
 	// metadata are filled for flow record. It is always true for Intra-Node
 	// and ToExternal flows and only applicable for Inter-Node flows that are
 	// not required to be correlated. (e.g. flows with Egress deny rule applied)
-	isMetaDataFilled bool
+	areCorrelatedFieldsFilled bool
+	// Some fields like Pod labels will be filled on the Flow Aggregator side,
+	// hasFilledExternalFields is an indicator to check whether these fields
+	// has been filled on Flow Aggregator.
+	hasFilledExternalFields bool
```

Comment:
I am amending my suggestion. I realized that we can have both of them in the same format: `areExternalFieldsFilled`. Please consider this modification.

Reply:
> I am amending my suggestion. I realized that we can have both of them in the same format: `areExternalFieldsFilled`. Please consider this modification.

Thanks, addressed. Please also take a look at https://github.com/vmware/go-ipfix/pull/212 Srikar, since we found limitation in current flowRecordMap data structure when adding these fields.